### PR TITLE
Added the onepassword:// url to the info field

### PR DESCRIPTION
**Pull request by rickfillion**
The intention of this is to have our clients read the pdf and find the onepassword url in the `onepassword` metadata field as opposed to having to parse text to get data out of it.

### CODE REVIEW

File: src/modules/draw.js [hunk 1]
```diff
@@ -154,6 +157,10 @@ function render(kit, drawFuncs) {
     return doc;
 }
 
+function onepasswordURL(kit) {
+    return "onepassword://team-account/add?email=" + encodeURI(kit.email) + "&key=" + encodeURI(kit.accountKey) + "&server=https://" + encodeURI(kit.teamURL);
```
Comment (kevinbhayes) [inline src/modules/draw.js]: In my testing kit.teamURL already has the https:// prefix, so I ended up with `https://https://account.b5test.com` in the subject.

**Comment by rickfillion** [comment]
You're right, @kevinbhayes. Fixed. thanks. :)

**Comment by galich** [comment]
Is it approved by @jpgoldberg ?
My concern is that we are about to give secret key away to OS indexers

**Comment by rickfillion** [comment]
I hadn't considered that, @galich . Thanks for bringing it up. 

@jpgoldberg I'd love for you to weigh in here. 

If I can try to make an argument for why this should be a non-issue... locally we don't consider the secret key to be a secret. Some OS indexers like Apple's Spotlight index not just the metadata of PDFs but also index the contents. And so this isn't a big change from that regard.

**Comment by rickfillion** [comment]
> What happens if you try adding a non-standard metadata property in the info object? Is it saved and can we still access it?

Good question. Let me try.

**Comment by robyoder** [comment]
Note: don't merge this PR yet. We're going to test this branch in a b5 branch before moving forward.

**Comment by rickfillion** [comment]
OK, we've tested this change on a B5 branch, and we've confirmed that both OPM and OPW are able to read the metadata. I believe this is ready to be merged now.

**Comment by rickfillion** [comment]
@jpgoldberg has told me that he won't have time to look  at this until next wednesday.

**Comment by galich** [comment]
oh wow, this is GitHub!
supporting info for @jpgoldberg :
* current *emergency kit.pdf* has secret key, email and url are obfuscated by ugly PDF ways. Simple text scanner won't see them. But it's not protected otherwise.
* custom property is actually a super simple to find text inside PDF file.

I believe it should not no difference to us.

**Comment by jpgoldberg** [comment]
Can someone clarify whether the current state of things is for a custom property or for the "subject" field?

File: src/modules/draw.js [hunk 2]
```diff
@@ -142,7 +142,10 @@ let drawFuncs = [
 function render(kit, drawFuncs) {
     let doc = new PDFDocument({
         margin: 0,
-        size: "letter"
+        size: "letter",
+        info: {
+            "OnePasswordURL": onepasswordURL(kit)
```
Comment (jpgoldberg) [inline src/modules/draw.js]: Ah. Cool. This is good and answers my question.

Comment (galich) [reply src/modules/draw.js]: not to stop or halt progress here, but worth to mention that this field actually in plain text inside PDF and it's actually easier to find compared to reading text from real PDF. You literally read PDF line by line (it's a text file) and see if line starts with OnePasswordURL

**Pull request merged**